### PR TITLE
Add new package: py-holland

### PR DESCRIPTION
**Pull request by darmac**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-holland/package.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack import *
+
+
+class PyHolland(PythonPackage):
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-holland/package.py]: There is already a `holland `library on PyPI and it's not this one. Let's name this package `py-holland-backup`

File: var/spack/repos/builtin/packages/py-holland/package.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    other database platforms and even non-database related applications.
+    Because of its plugin structure, Holland can be used to backup anything
+    you want by whatever means you want."""
+
+    homepage = "http://hollandbackup.org/"
+    url      = "https://github.com/holland-backup/holland/archive/1.2.2.tar.gz"
+
+    version('1.2.2', sha256='836337c243b2dff5ff6a3ce0b647f123ab24697a5de8ac8ae8b7839aa23dff68')
+
+    depends_on('py-setuptools', type='build')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-holland/package.py]:
```suggestion
    depends_on('py-setuptools', type=('build', 'run'))
```
It's listed in `install_requires`, so there's a good chance that the package installs some scripts that import `pkg_resources`

File: var/spack/repos/builtin/packages/py-holland/package.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    depends_on('py-configobj', type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-holland/package.py]:
```suggestion
    depends_on('py-configobj@4.6.0:', type=('build', 'run'))
```

**Review by adamjstewart — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/py-holland/package.py (hunk 3 above)
Comment (darmac) [reply var/spack/repos/builtin/packages/py-holland/package.py]: Fixed in https://github.com/spack/spack/pull/19924/commits/3b392269629a62cb0b5e9b8868afe5a6d8cea012

File: var/spack/repos/builtin/packages/py-holland/package.py (hunk 2 above)
Comment (darmac) [reply var/spack/repos/builtin/packages/py-holland/package.py]: Fixed in https://github.com/spack/spack/pull/19924/commits/46f46ab39d18cc0ec7c4e56fdbffaa4e5f008e63

File: var/spack/repos/builtin/packages/py-holland/package.py (hunk 1 above)
Comment (darmac) [reply var/spack/repos/builtin/packages/py-holland/package.py]: OK.

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**